Packs/CaseManagement-Generic/Classifiers/classifier-Case_Management_-_Generic_Mapper.json

```
@@ -0,0 +1,19 @@
+{
+    "description": "This mapper will set the labels for all Incidents created using it for either the system default Incident type, or the Case Type.   Can be used to quickly ingest alerts from any source without any specific mapping.",
+    "feed": false,
+    "id": "Case Management - Generic Mapper",
+    "mapping": {
+        "Case": {
+            "dontMapEventToLabels": false,
+            "internalMapping": null
+        },
+        "dbot_classification_incident_type_all": {
+            "dontMapEventToLabels": false,
+            "internalMapping": null
```

```suggestion
            "internalMapping": {}
        },
        "dbot_classification_incident_type_all": {
            "dontMapEventToLabels": false,
            "internalMapping": {}
```
this should solve the build issue. @beauchompers 
1. did you write it manually or was it auto-generated?
2. what is the purpose of having a mapper without mapping data?

---

It was auto-generated via the UI.